### PR TITLE
Development

### PR DESCRIPTION
**Pull request by sihrc**
- [x] version tick

### CODE REVIEW

File: indicoio/images/features.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from indicoio.utils.image import image_preprocess
 from indicoio.utils.api import api_handler
 
-def facial_features(image, cloud=None, batch=False, api_key=None, **kwargs):
+def facial_features(image, cloud=None, batch=False, api_key=None, version=None, **kwargs):
```
Comment (anneCarlson) [inline indicoio/images/features.py]: Don't we have a `detect` argument that can be passed as well?

Comment (sihrc) [inline indicoio/images/features.py]: It should be captured in **kwargs

Comment (sihrc) [inline indicoio/images/features.py]: The exposed kwargs to exposed functions are ones that aren't arguments for the API itself but rather routing / logistic arguments. Like.. cloud, batch, api, version all don't go into the predict function itself.

Comment (anneCarlson) [inline indicoio/images/features.py]: Ah!

File: indicoio/images/features.py [hunk 2]
```diff
@@ -26,17 +26,17 @@ def facial_features(image, cloud=None, batch=False, api_key=None, **kwargs):
     :rtype: List containing feature responses
     """
     image = image_preprocess(image, batch=batch, size=(48,48))
```
Comment (anneCarlson) [inline indicoio/images/features.py]: Do we want to specify 48x48 given the user could be setting the `detect` flag.

Comment (sihrc) [inline indicoio/images/features.py]: ? detect is for FER

Comment (sihrc) [inline indicoio/images/features.py]: oh. we also have detect for facialfeatuers :o.

Comment (madisonmay) [inline indicoio/images/features.py]: Also for facial features, technically.

Comment (anneCarlson) [inline indicoio/images/features.py]: So, thoughts on not resizing if the `detect` flag is set?

**Pull request merged**